Library/Formula/boost.rb

```diff
@@ -36,6 +36,10 @@ class Boost < Formula
     cause "Dropped arguments to functions when linking with boost"
   end
 
+  # Patch for odeint. Can (hopefully) be removed with the next release of Boost (1.58).
+  # See https://github.com/headmyshoulder/odeint-v2/issues/138
+  patch :DATA
```

Comment:
I can add more details to the comment. Also, the upstream project (odeint) _has_ incorporated this change. I don't know why it hasn't made it into Boost yet. I linked to the issue. You want me to link to the [fixing commit](https://github.com/headmyshoulder/odeint-v2/commit/0ca187cd6e83ddbaa5eedaa5d0d570986f351263) instead?